metadata/2025-10-02-disallow-init-module-to-publish-modules/disallow_init_module_to_publish_modules.json

```diff
@@ -0,0 +1,6 @@
+{
+  "title": "Disallow init module to attempt to publish modules",
+  "description": "This feature improves module publishing developer experience, amd is required for V2 loader",
```

Comment:
"amd is required for V2 loader" could you please explain a bit more on this?

Reply:
this is a typo right, this should be "and" @georgemitenkov ? :)